docs/tutorials/multimodal/convert_data_to_coco_format.md

```diff
@@ -0,0 +1,169 @@
+# Convert Data to MSCOCO Format
```

Comment:
We shall be consistent in the naming (among tutorials). I think COCO might be better.

Reply:
Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0